scripts/base/protocols/ldap/__load__.zeek

```diff
@@ -0,0 +1,4 @@
+@if ( have_spicy_analyzers() )
```

Comment:
What would you think about doing `@ifdef ( Analyzer::ANALYZER_LDAP )` (or the _UDP / _TCP versions) here?

I've done that over on the QUIC side, because there's a possibility of a too old OpenSSL version. That is likely not the case for LDAP, but still wondering from the consistency side.

Reply:
I'd be in favor of that, but we have two analyzers here and I cannot come up with a form to combine `@ifdef` and `||`. That means we'd either need to check only for one analyzer, or have nested `@ifed`s, both of which are nasty enough that they'd warrent a comment much more complicated than `@if ( have_spicy_analyzers() )`.